### PR TITLE
Card editor logic & back-end

### PR DESCRIPTION
**Pull request by vladimirDobrik**

### CODE REVIEW

**Comment by vladimirDobrik** [comment]
[bind card editor to new back-end logic](https://trello.com/c/2DFkYGze/58-bind-card-editor-to-new-back-end-logic)

**Comment by vladimirDobrik** [comment]
[add put method to back-end](https://trello.com/c/OMUoPw7o/59-add-put-method-to-back-end)

File: client/src/store/cards/cards.effects.ts [hunk 1]
```diff
@@ -12,33 +12,65 @@ import * as skillsActions from '../skills/skills.action';
 
 import * as cardActions from './cards.action';
 
+import { baseUrl } from './../../constants/baseUrl';
```
Comment (vharadkou) [inline client/src/store/cards/cards.effects.ts]: relative path

File: server/src/service/card/card.repository-implementation.ts [hunk 2]
```diff
@@ -80,4 +80,18 @@ export class CardRepositoryImplementation implements CardRepository {
             });
         });
     }
+
+    public updateCard(updatedCard: Card): Promise<boolean> {
+        return new Promise<boolean>((resolve, reject) => {
+            this.cardModel.updateOne({ id: updatedCard.id }, updatedCard, error => {
+                if(error) {
+                    reject(error);
+                    this.loggerService.errorLog(error);
```
Comment (vharadkou) [inline server/src/service/card/card.repository-implementation.ts]: log should be before reject

File: server/src/service/card/card.repository-implementation.ts [hunk 3]
```diff
@@ -80,4 +80,18 @@ export class CardRepositoryImplementation implements CardRepository {
...
+                } else {
+                    resolve(true);
+                    this.loggerService.infoLog('Successful updating card');
```
Comment (vharadkou) [inline server/src/service/card/card.repository-implementation.ts]: log should be before resolve

**Pull request merged**